contrib/go/src/python/pants/contrib/go/tasks/go_compile.py

```diff
@@ -60,9 +60,25 @@ def execute(self):
           lib_binary_map[vt.target] = os.path.join(gopath, 'pkg', self.goos_goarch,
                                                    vt.target.import_path + '.a')
 
+  @classmethod
+  def _split_build_flags(cls, build_flags):
```

Comment:
I'm fine with it since it is in-fact tested and that serves as useful enough doc to hopefully save others time grokking this. That said, `# Visible for testing` is probably better than `"""Visible for testing"""` since the triple quoted string implies API docs, and this is just a note to devs about a method already _private anyhow.